src/main/java/gov/cms/madie/services/HqmfService.java

```diff
@@ -16,12 +18,27 @@ public class HqmfService {
   private final SimpleXmlService simpleXmlService;
   private final HQMFGeneratorFactory hqmfGeneratorFactory;
 
-  public String generateHqmf(QdmMeasure qdmMeasure) throws Exception {
-    Generator hqmfGenerator = hqmfGeneratorFactory.getHQMFGenerator();
-    String simpleXml = simpleXmlService.measureToSimpleXml(qdmMeasure);
-    MeasureExport measureExport =
-        MeasureExport.builder().measure(qdmMeasure).simpleXml(simpleXml).build();
+  public String generateHqmf(QdmMeasure qdmMeasure) {
+    String simpleXml;
+    try {
+      simpleXml = simpleXmlService.measureToSimpleXml(qdmMeasure);
+    } catch (JAXBException ex) {
+      String message =
+          "An issue occurred while generating the simple xml for measure: " + qdmMeasure.getId();
+      log.error(message, ex);
+      throw new PackagingException(message, ex);
+    }
 
-    return hqmfGenerator.generate(measureExport);
+    Generator hqmfGenerator = hqmfGeneratorFactory.getHQMFGenerator();
+    try {
```

Comment:
Do you think we can use a single Try block but with multiple catch blocks ? Looks like the Exceptions we are catching are different, so I believe we can have multiple catch blocks

Reply:
we could, updated.